### ISSUE TITLE
[Bug] IndexError: index 1 is out of bounds 

### ISSUE THREAD

**Issue by eminfan** [deezer/spleeter#415]
Dear Sirs,

Some songs can not be spleted after  I installed the new Spleeter. Pls see below:

```

Traceback (most recent call last):
  File "/root/anaconda3/bin/spleeter", line 10, in <module>
    sys.exit(entrypoint())
  File "/root/anaconda3/lib/python3.7/site-packages/spleeter/__main__.py", line 54, in entrypoint
    main(sys.argv)
  File "/root/anaconda3/lib/python3.7/site-packages/spleeter/__main__.py", line 46, in main
    entrypoint(arguments, params)
  File "/root/anaconda3/lib/python3.7/site-packages/spleeter/commands/separate.py", line 45, in entrypoint
    synchronous=False
  File "/root/anaconda3/lib/python3.7/site-packages/spleeter/separator.py", line 191, in separate_to_file
    sources = self.separate(waveform, audio_descriptor)
  File "/root/anaconda3/lib/python3.7/site-packages/spleeter/separator.py", line 157, in separate
    return self.separate_librosa(waveform, audio_descriptor)
  File "/root/anaconda3/lib/python3.7/site-packages/spleeter/separator.py", line 145, in separate_librosa
    stft = self.stft(waveform)
  File "/root/anaconda3/lib/python3.7/site-packages/spleeter/separator.py", line 126, in stft
    dl, dr = (data[:, :, 0].T, data[:, :, 1].T) if inverse else (data[:, 0], data[:, 1])
IndexError: index 1 is out of bounds for axis 1 with size 1
```

## Environment

| OS                |  Ubuntu |
| Installation type | Conda          |
| Hardware spec     | CPU            |

**Comment by esketch9** [comment]
i'd same issue. lol

**Comment by mmoussallam** [comment]
Hi @eminfan 

we can't really help you with that little info. Is it only for some songs ? what command are you running ? are your song mono ?

**Comment by eminfan** [comment]
> Hi @eminfan
> 
> we can't really help you with that little info. Is it only for some songs ? what command are you running ? are your song mono ?

Thank you for your reply. Yes, it's only for some songs. 
cmd is "spleeter separate -i /opt/upload/" + id + "." + suffix + " -p spleeter:" + amount + "stems -o /opt/songs/ -c mp3"
I do not konw if it is mono. If you need it, I can upload to internet.
I downgrade spleeter to 1.4.9 , and then it succeeded.

**Comment by esketch9** [comment]
Thank you reply. 
16k, PCM, Mono audio file (wave and MP3), and even though the files that were good when they were two weeks ago were re-run, there is a problem.

**Comment by romi1502** [comment]
What version were you using when the issue appeared? was it 1.5.0?
A bug that resulted in error when processing mono files was introduced in 1.5.0 and fixed in 1.5.1, so we'd be interested if there is still such a bug in more recent version.
